unittests/testattachments.py

```diff
@@ -49,6 +53,96 @@ def test_attachments_str(self):
             msg="Incorrect str",
         )
 
+
+# A bug in the python mock_open helper function prevents the successful
```

Comment:
This is a very helpful comment. Please do something like this for the "must fix this" comment higher up the PR